test/test-server.js

```diff
@@ -200,7 +200,8 @@ tap.test('server test', function(t) {
         cmd: 'started',
         appName: 'test-app',
         agentVersion: '1.0.0',
-        pid: 1234
+        pid: 1234,
+        startTime: +(new Date())
```

Comment:
Turns out I had already made this change in the docker branch and just merged that change as part of a cleanup PR, so you'll have to rebase and can probably get rid of this hunk in the process anyway.